server/src/main/scala/org/elasticmq/server/config/ElasticMQServerConfig.scala

```diff
@@ -68,47 +71,75 @@ class ElasticMQServerConfig(config: Config) extends Logging {
 
   val restStatisticsConfiguration = new RestStatisticsConfiguration
 
+  private val queuesStorage = config.getConfig("queues-storage")
+  val queuesStoragePath: String = queuesStorage.getString("path")
+  val queuesStorageEnabled: Boolean = queuesStorage.getBoolean("enabled")
+
+  def readQueuesToLoad(persistedQueues: Seq[CreateQueue]): Seq[CreateQueue] = {
+    val persistedQueuesName = persistedQueues.map(_.name)
+    val baseQueues = createBaseQueues.filterNot(queue => persistedQueuesName.contains(queue.name))
+    persistedQueues ++ baseQueues
+  }
+
+  val createBaseQueues: Seq[CreateQueue] = {
+    val baseQueues: mutable.Map[String, ConfigValue] = config
+      .getObject("queues")
+      .asScala
+    createQueuesFromConfig(baseQueues)
+  }
+
+  private val persistedQueuesConfig: Option[Config] =
```

Comment:
let's keep a single name for the stored queues. Currently we have:
* queue storage
* persisted queues
* backup file

Let's keep one of: storage, persistence, backup